docs/user/user/pings/data-control-plane/product-integration.md

```diff
@@ -0,0 +1,8 @@
+# Product Integration
+
+Glean provides a general Nimbus feature named `glean` that can be used for configuration of pings. Simply select the `glean` feature along with your Nimbus feature from the Experimenter UI when configuring the experiment or rollout (see https://experimenter.info for more information on multi-feature experiments).
+
+The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the require metric configuration. The format of the configuration is defined in the [Experimenter Configuration](./experimenter-configuration.md) section. If a ping is not included, it will default to the value found in the pings.yaml.
```

Comment:
Not directly, internal pings disabled with the builtin_pings switch would still be diabled, but you could use Server Knobs PE (Pings Edition) to override that setting

Reply:
Okay, so we might want to throw a comment to that effect here. Though if SKPE (pronounced "skippy") doesn't have a use case to actually be used against builtins... maybe we forbid that? ...I guess it's a fair thing to leave in, in retrospect, since we might want to turn of "baseline" and might as well use SKPE to do it at the speed of Nimbus while waiting for `disable_builtin_pings` to ride trains...